workshops/filters.py

```diff
@@ -207,6 +211,139 @@ def get_order_by(self, order_value):
         return super().get_order_by(order_value)
 
 
+def filter_all_persons(queryset, all_persons):
+    if all_persons:
+        return queryset
+    else:
+        return queryset.filter(
+            task__role__name='learner',
+            task__event__tags__name='TTT').distinct()
+
+
+def filter_trainee_name(queryset, name):
+    return queryset.filter(Q(personal__icontains=name) |
```

Comment:
Wouldn't searching for "Wilson Banaszkiewicz" return empty record with this code? Other thing is that I don't know if I want such query to return anything…

So: I'm not 100% certain this query will do.